qi-doc/scribblings/qi.scrbl

```diff
@@ -40,6 +40,7 @@ This site hosts @emph{user} documentation. If you are interested in contributing
 @include-section["tutorial.scrbl"]
 @include-section["interface.scrbl"]
 @include-section["forms.scrbl"]
+@include-section["list-operations.scrbl"]
```

Comment:
This isn't actually added until the next commit (474b062 (Add preliminary
qi/list scribblings and fix bindings reference in qi/list module., 2024-06-14)),
so this change doesn't belong here.

Reply:
Addressed during rebases.